code/go/0chain.net/blobbercore/handler/protocol.go

```diff
@@ -83,7 +83,9 @@ func RegisterBlobber(ctx context.Context) (string, error) {
 	for _, sRegisteredNode := range sRegisteredNodes {
 		if sn.ID == string(sRegisteredNode.ID) || sn.BaseURL == sRegisteredNode.BaseURL {
 			Logger.Info("Failed during registering blobber to the mining network, it's duplicated")
```

Comment:
Looks like this workaround might stick around. Maybe it'd make sense to update the log to reflect what's happening after the change. Something like, "blobber already registered, updating settings".